modules/internal/ChapelIteratorSupport.chpl

```diff
@@ -42,6 +42,8 @@ module ChapelIteratorSupport {
   private use ChapelStandard;
   private use Reflection;
 
+  extern proc printf(s...);
```

Comment:
leftover debug? (btw there is chpl_debug_writeln that I find easier to use for debugging)

Reply:
Ugh... somehow, I couldn't get used to it.